frontend/public/locales/en/public.json

```diff
@@ -1406,13 +1406,12 @@
   "Show YAML": "Show YAML",
   "Samples": "Samples",
   "Snippets": "Snippets",
-  "OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.": "OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.",
-  "OpenShift helps you quickly develop, host, and scale applications. To get started, you'll need a project. Currently, you can't create or access any projects.": "OpenShift helps you quickly develop, host, and scale applications. To get started, you'll need a project. Currently, you can't create or access any projects.",
-  "You'll need to contact a cluster administrator for help.": "You'll need to contact a cluster administrator for help.",
-  "To learn more, visit the OpenShift <2>documentation</2>.": "To learn more, visit the OpenShift <2>documentation</2>.",
-  "Download the <1>command-line tools</1>": "Download the <1>command-line tools</1>",
   "Create a new project": "Create a new project",
-  "Getting Started": "Getting Started",
+  "Download command-line tools": "Download command-line tools",
+  "Hello, world": "Hello, world",
+  "To get started, create a project for your application.": "To get started, create a project for your application.",
+  "To get started, you'll need a project. Currently, you can't create or access any projects.": "To get started, you'll need a project. Currently, you can't create or access any projects.",
+  "You'll need to contact a cluster administrator for help.": "You'll need to contact a cluster administrator for help.",
```

Comment:
@logonoff @Leo6Leo I only found the   translations for `Currently, you can't create or access any projects.` in main branch. I couldn't find the others in full text search. I may be wrong!